lib/sw-precache.js

```diff
@@ -181,7 +181,7 @@ function generate(params, callback) {
     var runtimeCaching;
     var swToolboxCode;
     if (params.runtimeCaching) {
-      var pathToSWToolbox = require.resolve('sw-toolbox/sw-toolbox.js');
+      var pathToSWToolbox = require.resolve('sw-toolbox/build/sw-toolbox.js');
```

Comment:
Ack, my bad. I read through the list of commits to decide what kind of version bump I needed for the latest release, and missed that this caused a breaking change.